ScoreCard muiv5 style engine

ektasawant opened the pull request:
<!-- If this pull request fixes an Issue, link it below. If not, you can remove the line below -->
Fixes 3058 .

<!-- Include a bulleted list summarizing the main changes you have made in this PR -->
#### Changes proposed in this Pull Request:
- 
- 
- 

<!-- Include screenshots if they will help illustrate the changes in this PR -->
#### Screenshots:

<!-- Instruction for PR reviewers, if more complicated than a simple yarn start -->
#### To Test:
- /brightlayer-ui-components/surface-components

ektasawant commented:
<img width="1792" alt="Screenshot 2022-05-02 at 12 41 28 PM" src="https://user-images.githubusercontent.com/10433274/166231013-f1ba268e-8cc2-48ce-8fb0-8cd5093f71f8.png">
Getting this error, please let me know if there is any sugesstion.

daileytj reviewed (approved):
@ektasawant I hope you don't mind me pushing directly to your branch, but I cleaned up some types and fixed that console error.

ektasawant commented:
> @ektasawant I hope you don't mind me pushing directly to your branch, but I cleaned up some types and fixed that console error.

Thanks @daileytj , for fixing this PR.
